roles/openshift_logging/README.md

```diff
@@ -20,20 +20,22 @@ When `openshift_logging_install_logging` is set to `False` the `openshift_loggin
 
 ### Optional vars:
 - `openshift_logging_purge_logging`: When `openshift_logging_install_logging` is set to 'False' to trigger uninstalation and `openshift_logging_purge_logging` is set to 'True', it will completely and irreversibly remove all logging persistent data including PVC. Defaults to 'False'.
-- `openshift_logging_image_prefix`: The prefix for the logging images to use. Defaults to 'docker.io/openshift/origin-'.
-- `openshift_logging_curator_image_prefix`: Setting the image prefix for Curator image. Defaults to `openshift_logging_image_prefix`.
-- `openshift_logging_elasticsearch_image_prefix`: Setting the image prefix for Elasticsearch image. Defaults to `openshift_logging_image_prefix`.
-- `openshift_logging_fluentd_image_prefix`: Setting the image prefix for Fluentd image. Defaults to `openshift_logging_image_prefix`.
-- `openshift_logging_kibana_image_prefix`: Setting the image prefix for Kibana image. Defaults to `openshift_logging_image_prefix`.
-- `openshift_logging_kibana_proxy_image_prefix`: Setting the image prefix for Kibana proxy image. Defaults to `openshift_logging_image_prefix`.
+- `openshift_logging_image_prefix`: The prefix for all of the logging images to use. The order in which prefix will be determined is 1) each component image has own default, 2) setting `openshift_logging_image_prefix` selects prefix for all components, 3) the prefix can be cherry-picked with `openshift_logging_{{component}}_image_prefix` for a specific component, eg. `openshift_logging_elasticsearch_image_prefix`.
+- `openshift_logging_curator_image_prefix`: Setting the image prefix for Curator image. Defaults to `__openshift_logging_image_prefix`.
```

Comment:
We probably should not be listing the defaults as the contents of another variable, it's a little obscure for users.
I'm also not sure we should be advertising the name of our private variables....

Reply:
ack, updated both READMEs